### ISSUE TITLE
problems downloading dependencies

### ISSUE THREAD

**Issue by egonw** [scijava/scyjava#42]
I observed last week a new problem: Maven Central dependencies are no longer loaded correctly. I first had this when making [a PR for pybacting](https://github.com/cthoyt/pybacting/pull/9#issuecomment-1173468987), but later also noted this with a [Jupyter notebook that worked fine before](https://colab.research.google.com/github/bioclipse/bioclipse.scripting/blob/master/nb/ParseIUPACName.ipynb):

![image](https://user-images.githubusercontent.com/26721/178136033-3c15ce47-af8a-4289-a713-ffadb940df6c.png)

How can I debug this? Is this a ScyJava problem? Did something change at Maven Central?

**Comment by egonw** [comment]
Mmm... I actually just now notice that [this notebook](https://colab.research.google.com/github/egonw/chempyformatics/blob/main/docs/nb/ReadSMILES.ipynb) does not have problems and also loads from Maven Central (but using the old API).

**Comment by egonw** [comment]
The following command (which underlies the resolving, I guess) works for me locally:

```shell
/usr/bin/mvn -B -f /home/egonw/var/Projects/hub/bacting/managers-cheminfo/net.bioclipse.managers.inchi/pom.xml dependency:resolve
```

**Comment by egonw** [comment]
Okay, problem is elsewhere (finally found it): https://github.com/eclipse-equinox/equinox.bundles/issues/54

**Comment by egonw** [comment]
Okay, I now depend on the latest Eclipse bundles which should depend on the fixed equinox library, but the the wrong version is tried by ScyJava: https://github.com/cthoyt/pybacting/issues/11

**Comment by imagejan** [comment]
@egonw is this still an issue? It might be a problem with `jgo` that keeps the outdated version in its cache. Did you try removing your `.jgo` folder and then running your script again?

**Comment by egonw** [comment]
@imagejan, I will test.

**Comment by egonw** [comment]
Yes, I still seem to have similar problems, but the equinox one seems to be solved. I find it really hard to debug. The latest (failing) PR is here: https://github.com/cthoyt/pybacting/pull/12

So, running/testing Bacting 0.0.44 itself works fine, but in Python (with ScyJava) it fails. Right now, Apache's `commons-io`. One thing that seems to work is explicitly excluding from depends and then explicitly added a dependency. But that did not seem to solve the `commons-io` issue yet. Again, I find it really hard to learn what to do from the (Maven) error messages. ScyJava does not make that easier (not its responsibility, really, of course).
